### PR TITLE
Units: Add events/messages/records/rows throughput units

### PR DESCRIPTION
**Pull request by hhromic**
**What is this feature?**

This commit adds support for additional throughput units to `valueFormats`:
- `events/sec (eps)` and `events/min (epm)`
- `messages/sec (mps)` and `messages/min (mpm)`
- `records/sec (rps)` and `records/min (rpm)`
- `rows/sec (rps)` and `rows/min (rpm)`

These new units are very useful for monitoring streaming, e.g. Kafka or RedPanda, and database systems.

I also added missing entries for existing throughput units to the dev dashboards tests.

**Why do we need this feature?**

Monitoring streaming and database systems with Prometheus and Grafana is very common. However, there are no adequate throughput units in Grafana for them. Streaming systems often use the concepts of "events", "messages" or "records" for entities that are processed. Similarly, databases have rows. None of these are available naitvely as a value format in Grafana.

This is a long standing missing feature (see referenced discussion).

**Who is this feature for?**

Any user that monitors streaming and/or database systems with Grafana. For example, users that build dashboard with throughput panels for event processing, or ETL processes loading data into databases.

**Which issue(s) does this PR fix?**:

Closes https://github.com/grafana/grafana/discussions/36999

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

Please let me know if documentation/release notes updates are needed for this change.

### CODE REVIEW

**Review by mckn — APPROVED** [review]
LGTM! Waiting for a second approval before moving this forward.

**Review by ivanortegaalba — APPROVED** [review]
👍

**Pull request merged**
